File: sdkwork-backend-sdk-typescript/src/types/page-plus-dns-record-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusDnsRecordVO } from './plus-dns-record-vo';
import type { SortObject } from './sort-object';

export interface PagePlusDnsRecordVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusDnsRecordVO[];
  number?: number;
  numberOfElements?: number;
  pageable?: PageableObject;
  sort?: SortObject;
  first?: boolean;
  last?: boolean;
  empty?: boolean;
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/page_plus_dns_record_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PagePlusDnsRecordVO:
    total_pages: int = None
    total_elements: int = None
    size: int = None
    content: List[PlusDnsRecordVO] = None
    number: int = None
    sort: SortObject = None
    first: bool = None
    last: bool = None
    number_of_elements: int = None
    pageable: PageableObject = None
    empty: bool = None
